### ISSUE TITLE
enable gateway in rc.conf fails

### ISSUE THREAD

**Issue by thallian** [ployground/bsdploy#56]
While running `ploy configure` I get an error at the `Enable gateway in rc.conf` step (using the latest master build):

```
gateway does not exist in /etc/rc.d or the local startup
directories (/usr/local/etc/rc.d)

msg: unable to determine rcvar
```

I am a complete beginner in FreeBSD (and in BSD systems in general) so my findings may be obvious to you.

The ansible service module tries to get the service status before enabling/disabling it and as there is no gateway file in `/etc/rc.d/` this fails.

This happens only after the recent change to [pf.yml](/ployground/bsdploy/blob/master/bsdploy/roles/jails_host/tasks/pf.yml) as seen in this [changeset](/ployground/bsdploy/commit/8a75ba0c6af8e8129751f13932fcd0a15c901b67).

After resetting to this commit everything works as described.

**Comment by tomster** [comment]
thanks @thallian for reporting this. had i known anybody is actually using this stuff i'd have pushed the fix sooner :-) (i ran into the issue myself)

it's now fixed in f82539abb885d2e184771f78b1510f87ecbe01ee

**Comment by thallian** [comment]
Perfect, thanks!  
Yeah, I am only testing for the moment ;)
